drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -1132,8 +1154,18 @@ local function device_added(driver, device)
     device:emit_event(capabilities.energyMeter.energy({ value = 0.0, unit = "Wh" }))
   end
 
-  -- call device init in case init is not called after added due to device caching
-  device_init(driver, device)
+  -- When device_init is performed in the aqara switch that has EDGE_CHILE,
+  -- a problem related to capability mapping occurs during configuration_buttons(), so this conditions are added.
+  if device.manufacturer_info.vendor_id ~= AQARA_MANUFACTURER_ID then
```

Comment:
Could this be simplified into the following?
```suggestion
  if not (device.manufacturer_info.vendor_id == AQARA_MANUFACTURER_ID and 
    (device.manufacturer_info.product_id == 0x1008 or device.manufacturer_info.product_id == 0x1009)) then
    -- call device init in case init is not called after added due to device caching
      device_init(driver, device)
 end
```